src/modules/users/add-user/add-user.html

```diff
@@ -43,7 +43,7 @@
             <div class="form-group">
                 <label class="col-sm-2 col-md-1 control-label" for="password">Password</label>
                 <div class="col-sm-8 col-md-6 inner-addon">
-                    <input type="{{ addUserCntrl.typePassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="password" ng-model="addUserCntrl.user.password" minlength="8" required/>
+                    <input type="{{ addUserCntrl.typePassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="password" ng-model="addUserCntrl.user.password" minlength="9" ng-maxlength="128" required/>
```

Comment:
if you have used ng-maxlength then why you have not used ng-minlength?

Reply:
Same as above comment.